changes.xml

```diff
@@ -24,6 +24,9 @@
   <body>
 
     <release version="1.6.0" date="not released">
+      <action type="add" dev="nbellack">
+        Role aem-dispatcher: Add http2 variant.
```

Comment:
this comment should replace the existing comment - http2 is no longer enabled by default